src/components/map.tsx

```diff
@@ -71,7 +71,7 @@ const CH_BBOX: BBox = [
 const constrainZoom = (
   viewState: $FixMe,
   bbox: BBox,
-  { padding = 150 }: { padding?: number } = {}
+  { padding = 20 }: { padding?: number } = {}
```

Comment:
Maybe we make padding required, this way it's easier to follow ? It seems we are passing it below, so no need for this additional default value ?